rabbitmq/check.py

```diff
@@ -294,16 +306,21 @@ def get_stats(self, instance, base_url, object_type, max_detailed, filters, cust
             # We truncate the list of nodes/queues if it's above the limit
             self._get_metrics(data_line, object_type, custom_tags)
 
-    def _get_metrics(self, data, object_type, custom_tags):
-        tags = []
-        tag_list = TAGS_MAP[object_type]
-        for t in tag_list:
-            tag = data.get(t)
-            if tag:
-                # FIXME 6.x: remove this suffix or unify (sc doesn't have it)
-                tags.append('%s_%s:%s' % (TAG_PREFIX, tag_list[t], tag))
-        tags.extend(custom_tags)
+        # get a list of the number of bindings on a given queue
+        # /api/queues/vhost/name/bindings
+        if object_type is QUEUE_TYPE:
```

Comment:
Since we only want to do this for `QUEUE_TYPE` we should move it to a specific function. This would make the code easier to read.